File: rb/utility/make_context.rb
# BreakingBadQuotes SDK utility: make_context
require_relative '../core/context'
module BreakingBadQuotesUtilities
  MakeContext = ->(ctxmap, basectx) {
    BreakingBadQuotesContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// BreakingBadQuotes SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class BreakingBadQuotesMakeContext
{
    public static function call(array $ctxmap, ?BreakingBadQuotesContext $basectx): BreakingBadQuotesContext
    {
        return new BreakingBadQuotesContext($ctxmap, $basectx);
    }
}
